docs/features/grpc-action.rst

```diff
@@ -4,8 +4,106 @@ GRPCAction
 Description
 -----------
 
+With Django-Socio-GRPC you can declare a grpc action related to your app service with the decorator grpc_action. 
+Once your service registered, it will create a new request and response in your .proto file after the proto generation.$
+
+The decorator signature is:
+.. code-block:: python
+
+    def grpc_action(request=None, response=None, request_name=None, response_name=None, request_stream=False, response_stream=False, use_request_list=False, use_response_list=False)
+
 Usage
 -----
 
-Example
--------
+======
+Import
+======
+
+First of all, you need to import grpc_action
+
+.. code-block:: python
+
+    from django_socio_grpc.decorators import grpc_action 
+
+This decorator can now be used for each action of your service.
+
+========================
+Request and Response arg
+========================
+
```

Comment:
Use protobuf message specs to indicate the behavior, you can write examples of different requests/response made into .proto messages